src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -64,6 +68,38 @@ public String[] LiteralPath
         public Stream InputStream { get; set; }
 
         /// <summary>
+        /// InputString parameter
+        /// The strings to calculate a hash
+        /// We allow `null` and `empty` strings because we can get it from pipeline.
+        /// </summary>
+        /// <value></value>
+        [Parameter(Mandatory = true, ParameterSetName = StringHashParameterSet, Position = 0, ValueFromPipeline = true)]
+        [AllowNull()]
+        [AllowEmptyString()]
+        public string[] InputString { get; set; }
+
+        /// <summary>
+        /// Encoding parameter
+        /// The Encoding of the 'InputString'
```

Comment:
This comment is confusing because a `string` is always encoded in `Unicode`.
A better comment might be something like:

> the 'InputString' converted to the encoding if necessary

I do wonder if this parameter makes sense. In light of the approach in #4119 - the parameter becomes `FileEncoding`, which then becomes very confusing for this cmdlet - it applies to the string and not the file.

Given how uncommon it is to hash strings in other encodings, maybe it's fine to require the caller to convert to another encoding and this cmdlet also support hashing a byte array.